account_reconcile_oca/README.rst

```diff
@@ -0,0 +1,96 @@
+=====================
+Account Reconcile Oca
+=====================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--reconcile-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-reconcile/tree/16.0/account_reconcile_oca
+    :alt: OCA/account-reconcile
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-reconcile-16-0/account-reconcile-16-0-account_reconcile_oca
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/98/16.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This addon allows to reconcile bank statements and account marked as `reconcile`.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+Bank reconcile
+~~~~~~~~~~~~~~
+
+Access `Invoicing / Dashboard` with a user with Full Acounting capabilities.
+Select reconcile on the journal of your choice.
+
+Account reconcile
+~~~~~~~~~~~~~~~~~
+
+Access `Invoicing / Accounting / Actions / Reconcile`
```

Comment:
In this view I see an empty list:

![image](https://user-images.githubusercontent.com/23449160/229154734-5920faf1-f981-407d-8d7b-eec8468395ba.png)

However from the banks journal it is fine:

![image](https://user-images.githubusercontent.com/23449160/229154817-80cf74d2-ccb6-481c-a168-fc99375c993e.png)

Reply:
The first one is empty as you don't have any accounts move lines to reconcile, Both widgets are similar but different, as the reconcile accounts is not able to change the move lines as the account.bank.statement